Dockerfile.docs

```diff
@@ -1,19 +1,18 @@
-FROM continuumio/anaconda3
+FROM ubuntu:17.10
```

Comment:
why not an LTS version?

Reply:
I could change it to LTS if you want. There was no reasoning behind using 17.10